ilimit.c

```diff
@@ -75,22 +76,27 @@ PHP_MINIT_FUNCTION(ilimit)
 {
     zend_class_entry ce;
 
+    INIT_NS_CLASS_ENTRY(ce, "ilimit", "Error\\LimitReached", NULL);
```

Comment:
`ilimit\Error` is probably a better base exception name ...

Reply:
In that case, should we keep the other errors under `ilimit\Error`?

Reply:
Yes